SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/EventTracingTests/ComponentUpdateEventTracingTest.cpp

```diff
@@ -7,7 +7,7 @@ AComponentUpdateEventTracingTest::AComponentUpdateEventTracingTest()
 	Author = "Matthew Sandford";
 	Description = TEXT("Test checking the component update trace events have appropriate causes");
 
-	FilterEventNames = { ComponentUpdateEventName, ReceiveOpEventName, MergeComponentUpdateEventName };
```

Comment:
How come the removal of `MergeComponentUpdateEventName`?

Reply:
MergeComponentUpdateEvent was an old event that does not exist any more.